src/effects/native/flangereffect.cpp

```diff
@@ -24,27 +24,18 @@ EffectManifest FlangerEffect::getManifest() {
         "A simple modulation effect, created by taking the input signal "
         "and mixing it with a delayed, pitch modulated copy of itself."));
 
-    EffectManifestParameter* delay = manifest.addParameter();
-    delay->setId("delay");
-    delay->setName(QObject::tr("Delay"));
-    delay->setDescription(QObject::tr("Sets the value for the delay length."));
-    delay->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
-    delay->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
-    delay->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
-    delay->setDefault(3333.3);
-    delay->setMinimum(50.0);
-    delay->setMaximum(10000.0);
-
     EffectManifestParameter* period = manifest.addParameter();
     period->setId("period");
     period->setName(QObject::tr("Period"));
-    period->setDescription(QObject::tr("Controls the speed of the effect."));
+    period->setDescription(QObject::tr("Controls the period of the LFO (low frequency oscillator)\n"
+        "1/4 - 4 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
+        "0.05 - 4 seconds if no tempo is detected (mic & aux inputs, master mix)"));
     period->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     period->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
-    period->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
-    period->setDefault(666666.6);
-    period->setMinimum(50000.0);
-    period->setMaximum(2000000.0);
+    period->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
+    period->setMinimum(0.00);
+    period->setMaximum(4.00);
+    period->setDefault(0.50);
```

Comment:
I prefer here 2. That sounds good and is the centre position. 
1 works also for me. (currently there is a by two bug that makes 0.50 to 1)